### PR TITLE
Javascript javascript3 week1

### PR DESCRIPTION
**Pull request by VioMNE**

### CODE REVIEW

File: javascript/javascript3/week1/weatherapp.js [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+      })
+      .then((data) => this.displayWeather(data));
+  },
+
+  displayWeather: function (data) {
+    const { name } = data;
+    const { icon, description } = data.weather[0];
+    const { temp, humidity } = data.main;
+    const { speed } = data.wind;
+    const { sunrise, sunset } = data.sys;
```
Comment (Enqira) [inline javascript/javascript3/week1/weatherapp.js:25]: Nice done here!

File: javascript/javascript3/week1/weatherapp.js [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+    console.log(
+      name,
+      icon,
+      description,
+      temp,
+      humidity,
+      speed,
+      sunrise,
+      sunset
```
Comment (Enqira) [inline javascript/javascript3/week1/weatherapp.js:34]: You could remove the console log before submiting

File: javascript/javascript3/week1/weatherapp.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+
+document
+  .querySelector(".search-bar")
+  .addEventListener("keyup", function (event) {
+    if (event.key === "Enter") {
+      weather.search();
+    }
+  });
+
+weather.fetchWeather("Copenhagen");
```
Comment (Enqira) [inline javascript/javascript3/week1/weatherapp.js:70]: impressive, it seems that you have at least a couple of years of experience with JS.

**Review by Enqira — APPROVED** [review]
Well done :)

File: javascript/javascript3/week1/weatherapp.html [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+<body>
+    <div class="card">
+        <div class="search">
+            <input type="text" class="search-bar" placeholder="Enter city"/>
+            <button><svg stroke="currentColor" fill="currentColor" stroke-width="0" viewBox="0 0 24 24" height="1.5em" width="1.5em" xmlns="http://www.w3.org/2000/svg"><path d="M10,18c1.846,0,3.543-0.635,4.897-1.688l4.396,4.396l1.414-1.414l-4.396-4.396C17.365,13.543,18,11.846,18,10 c0-4.411-3.589-8-8-8s-8,3.589-8,8S5.589,18,10,18z M10,4c3.309,0,6,2.691,6,6s-2.691,6-6,6s-6-2.691-6-6S6.691,4,10,4z"></path></svg></button>
+        </div>
+        <div class="weather">
+            <h2 class="city">Weather in Copenhagen</h2>
+            <div class="felx"></div>
+            <h1 class="temp">12°c</h1>
```
[the remark this answers is not in the document]
Comment (VioMNE) [reply javascript/javascript3/week1/weatherapp.html:23]: I used it just becasuse of the bigger font and it looks nicer for me to have H2 for city and H1 for temp :) is it ok if i leave it that way tho?

File: javascript/javascript3/week1/weatherapp.js (hunk 3 above)
Comment (VioMNE) [reply javascript/javascript3/week1/weatherapp.js:70]: Thank you for your input! :)

**Pull request merged**